src/app/network/crypto.test.ts

```diff
@@ -0,0 +1,82 @@
+/**
+ * @jest-environment jdom
+ */
+
+import { describe, expect, it, vi } from 'vitest';
+import { encryptFilename, generateHMAC, getEncryptedFile, processEveryFileBlobReturnHash } from './crypto';
+import { getHmacSha512 } from '../crypto/services/utils';
+import { Buffer } from 'buffer';
+import crypto from 'crypto';
+
+describe('Test crypto.ts functions', () => {
+  globalThis.Buffer = Buffer;
+  it('encryptFilename should generate a ciphertext', async () => {
+    const mnemonic =
+      'until bonus summer risk chunk oyster census ability frown win pull steel measure employ rigid improve riot remind system earn inch broken chalk clip';
+    const bucketId = 'test busket id';
+    const filename = 'test filename';
+    const result = await encryptFilename(mnemonic, bucketId, filename);
+    expect(result).toBeDefined();
```

Comment:
dummy test, but helps coverage. There is no corresponding decrypt function as it's a legacy algorithm used in cross-checking different platforms (?)